src/components/Profile/components/PersonalInfoList/index.jsx

```diff
@@ -369,14 +369,20 @@ const PersonalInfoList = ({ myProf, profile, isOnline, createSnackbar }) => {
             <Grid container item xs={5} alignItems="center">
               <Typography>{'Mailbox:'}</Typography>
             </Grid>
-            <Grid container item xs={myProf && mailCombo ? 2.5 : 5} alignItems="center">
+            <Grid
+              container
+              item
+              xs={myProf && mailCombo ? 2.5 : 5}
+              lg={myProf && mailCombo ? 2.4 : 5}
+              alignItems="center"
+            >
               <Typography>{`#${profile.Mail_Location}`}</Typography>
             </Grid>
             {myProf && mailCombo && (
               <>
-                <Grid container item xs={1} alignItems="center">
+                <Grid container item xs={1.1} alignItems="center">
                   <Typography className={styles.private}>
-                    {showMailCombo ? mailCombo : '****'}
+                    {showMailCombo ? '12 12 12' : '****'}
```

Comment:
```suggestion
                    {showMailCombo ? '12 12 12' : '****'}
```
why 12 12 12?

Reply:
I was showing a demo to someone and I forgot to remove it after showing the demo, I'll fix it